### PR TITLE
Added de locale for qwertz-keyboards

### PR DESCRIPTION
**Pull request by bernikr**

### CODE REVIEW

**Comment by Zhouzi** [comment]
Thank you @bernikr! Glad to add support for german keyboards. Would you mind translating the demo dialogue? It'd be much appreciated :)

File: build/locales/theater.de.js [hunk 1]
```diff
@@ -0,0 +1 @@
+(function (w) { w.TheaterJS.prototype.keyboards.de = ["qwertzuiopü", "asdfghjklöä", "yxcvbnm"]; })(window);
```
Comment (Zhouzi) [inline build/locales/theater.de.js]: This file is supposed to be in the `src/locales/` folder. Gulp's job is to build the appropriate files automatically. If you put this file in `src/locales/` and run `gulp`, it'll create both `theater.de.js` and `theater.de.min.js`.

**Comment by bernikr** [comment]
I did not realize you were using gulp... It should be fixed now. Should I make a second pull request for the translation or commit it to this one?

**Comment by Zhouzi** [comment]
Simply commit it to this one, I'll merge then :)

**Comment by Zhouzi** [comment]
Thank you @bernikr, it has been merged :)

**Pull request merged**